### PR TITLE
[ONNX] Extend ONNX Frontend with BitwiseNot-18 operator #20793

### PR DESCRIPTION
**Pull request by rupeshs**
### Details:
 - Added BitwiseNot operator to ONNX FrontEnd

### Tickets:
 - 118961

### CODE REVIEW

File: src/bindings/python/tests_compatibility/test_onnx/test_backend.py [hunk 1]
```diff
@@ -526,9 +526,6 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
         "OnnxBackendNodeModelTest.test_bitwise_and_i16_3d_cpu",
         "OnnxBackendNodeModelTest.test_bitwise_and_i32_2d_cpu",
         "OnnxBackendNodeModelTest.test_bitwise_and_ui64_bcast_3v1d_cpu",
-        "OnnxBackendNodeModelTest.test_bitwise_not_2d_cpu",
-        "OnnxBackendNodeModelTest.test_bitwise_not_3d_cpu",
-        "OnnxBackendNodeModelTest.test_bitwise_not_4d_cpu",
```
Comment (mitruska) [inline src/bindings/python/tests_compatibility/test_onnx/test_backend.py:531]: There are two similar files for the new and compatibility API tests. 
Could you please make the same change also in the `src/frontends/onnx/tests/tests_python/test_backend.py` file:
https://github.com/openvinotoolkit/openvino/blob/f3fc5032880aac72565466ca8e207bb211398685/src/frontends/onnx/tests/tests_python/test_backend.py#L397-L399

**Review by mitruska — COMMENTED** [review]
Thank you for this contribution! Overall looks good, let's enable few more tests.

File: src/bindings/python/tests_compatibility/test_onnx/test_backend.py (hunk 1 above)
Comment (rupeshs) [reply src/bindings/python/tests_compatibility/test_onnx/test_backend.py:531]: @mitruska  Thanks for the review, removed those in the latest commit.

**Comment by mlukasze** [comment]
build_jenkins

**Comment by rupeshs** [comment]
@mitruska could you please take a look at this?
Thanks!

**Comment by gkrivor** [comment]
Hi @rupeshs , thank you for contribution! Could you rebase your change?

Also I may expect you will need to put uint64 test into list of skipped tests: https://github.com/openvinotoolkit/openvino/blob/36d0b69b50f8640fd82aa636560f730cb5a4ccb0/src/frontends/onnx/tests/tests_python/test_backend.py#L413

**Pull request closed without merging**